examples/test.g

```diff
@@ -3,7 +3,7 @@ LoadPackage( "Grape" );
 LoadPackage( "Digraph" );
 Petersen := Graph( SymmetricGroup(5), [[1,2]], OnSets,
                    function(x,y) return Intersection(x,y)=[]; end );
-Petersen := Digraph( Petersen );
+Petersen := Digraph( Petersen );;
```

Comment:
What's the motivation for this?

Reply:
I don't recall. Maybe I tried to fix problems that arose due to the depiction of the Digraph-package? I reverted that change.